minimal-plugin/src/testFixtures/groovy/io/micronaut/gradle/AbstractGradleBuildSpec.groovy

```diff
@@ -17,6 +19,13 @@ abstract class AbstractGradleBuildSpec extends Specification {
         return GraalUtil.isGraalJVM() || System.getenv("GRAALVM_HOME")
     }
 
+    boolean containsDependency(String mavenCoordinate, String configuration) {
+        BuildResult result = build('dependencies')//  "--configuration ${configuration}"
```

Comment:
:x: Looks like the `configuration` parameter is unused

Reply:
I was hoping you could help me out. `AbstractGradleBulidSpec` throws `> Unknown command-line option '--configuration compileClasspath'.` if I uncomment it.  I was expecting to be able to use `"dependencies", "--configuration compileClasspath"` as a parameter.